tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -2370,6 +2370,10 @@ Count_Person_InLaborForce_ResidesInCollegeOrUniversityStudentHousing,Number of L
 Count_Person_InLaborForce_ResidesInGroupQuarters,Number of Labor Force Participants reside in Group Quarters
 Count_Person_InLaborForce_ResidesInNoninstitutionalizedGroupQuarters,Number of Labor Force Participants reside in Noninstitutionalized Group Quarters
 Count_Person_IsInternetUser_PerCapita,percentage of internet users
+Count_Person_InvolvedInCrash_Motorists,Number of motorists involved in crash
+Count_Person_InvolvedInCrash_MotorVehicleOccupant,Number of motor vehicle occupants involved in crash
+Count_Person_InvolvedInCrash_NonMotorists,Number of non-motorists involved in crash

```

Comment:
right now we don't add "non xxx" and "other xxx" variables to the index since they are vague when matching query (or even match the opposite meaning)